File: 프로그래머스/unrated/120892. 암호 해독/암호 해독.java
class Solution {
    public String solution(String cipher, int code) {
        String answer = "";

        for (int i = code - 1; i < cipher.length(); i += code) {
            answer += cipher.charAt(i);
        }

        return answer;
    }
}

// 다른 문제 풀이 방법

class Solution {
    public String solution(String cipher, int code) {
        String answer = "";
        String[] split = cipher.split("");

        for (int i = 0; i < split.length; i++) {
            if((i+1)% code ==0){
                answer += split[i];
            }
        }

        return answer;
    }
}

File: 프로그래머스/unrated/120892. 암호 해독/암호 해독.js
function solution(cipher, code) {
   let answer = "";
    let split = cipher.split("");
    
    for (let i = 0; i < split.length; i++) {
        if ((i+ 1) % code === 0) {
            answer += split[i]
        }
    }

    return answer;
}